### DISCUSSION TITLE
After Wifi Configuration SoftAp still exist

### DISCUSSION THREAD

**Discussion by joko2020** [Suraj151/pdi-framework#6] [Q&A]
Halo @Suraj151 , I Just want to asking about behavior of wifimanager library & ewingstack , I try the second time of the esp8266 stack and the problem is after configuration to my wifi local network, the default esp8266stack SoftAp still exist, it is different behavior when I use wifimanager library , the SoftAp will be disappear after success connecting to the local wifi network.
how to config ewingstack to act like wifimanager library.

Thankyou

**Comment by Suraj151** [comment]
Hello @joko2020 
the default behaviour of ewingstack is that it will keep wifi to work in both mode i.e. wifi stration and wifi access point mode, so even if you configure esp to your local network, soft ap will still be available. whether you use or not use the soft ap but it will always be available to use if needed. it will not affect your any other things so no worry.
this is required in most of the application. if you want to make it off you can do it in wifi init sequence where we use to configure access point.

**Comment by joko2020** [comment]
> Hello @joko2020
> the default behaviour of ewingstack is that it will keep wifi to work in both mode i.e. wifi stration and wifi access point mode, so even if you configure esp to your local network, soft ap will still be available. whether you use or not use the soft ap but it will always be available to use if needed. it will not affect your any other things so no worry.
> this is required in most of the application. if you want to make it off you can do it in wifi init sequence where we use to configure access point.

Thanks for your information, after i trial for several time , I noticed that my esp8266 become a router when I use Ewingstack and joining my local network, I mean i still able to use Internet with different subnet (my ip come from esp8266 not from my local Wilan).


> if you want to make it off you can do it in wifi init sequence where we use to configure access point.

ok I'll Try

Thanks again

**Comment by Suraj151** [comment]
Yes.... your esp8266 will act as router here. And since we kept napt feature enabled by default, it will also share the network/internet to its own soft ap local network from the station to which it is connected. 

There is one dynamic subnetting feature provided with this stack with which you can achieve autonomous network connections based on internet availability in between group of esp devices. If your application need then you can enable that feature as well.
You can refer => https://github.com/Suraj151/esp8266-framework/wiki/Common-Configs#toggle-subnetting-disabled-by-default
